pages/nfc/_landing.vue

```diff
@@ -0,0 +1,190 @@
+            </p>
+          </div>
+        </div>
+
+        <img src="/img/divider.png" class="my-1" />
+
+        <div class="flex items-center">
+          <TIcon name="card-network" />
+          <div class="ml-2">
+            <h3 class="text-slate-700 text-bold text-lg">Start networking</h3>
+            <p class="text-gray-500 text-sm">
+              Share your profile by tapping your VIP Card on other
+              people&rsquo;s mobiles and start networking faster
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <TBenefits
+      title="Premium Membership"
+      icon="check-green"
+      :benefits="[
+        'Get special offers from organisers',
+        'Exclusive discounts',
+        'Access to online classes',
+      ]"
+    />
+
+    <section class="max-w-md m-auto h-64 p-3 mt-8">
+      <h2 class="text-3xl text-center font-extrabold mb-2">Follow us</h2>
+      <p class="font-normal text-lg text-center">
+        We regularly post event announcements and introduce new members on our
+        social media.
+      </p>
+      <div class="max-w-md m-auto pb-16">
+        <div class="flex justify-center gap-3 pt-8">
+          <a href="https://www.instagram.com/WeDancePlatform/" target="_blank">
+            <TIcon
+              name="instagram"
+              size="10"
+              class="rounded-full p-2"
+              style="box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1), 0px 1px 2px rgba(0, 0, 0, 0.06);"
+            />
+          </a>
+          <a href="https://t.me/WeDanceTravel" target="_blank">
+            <TIcon
+              name="telegram"
+              size="10"
+              class="rounded-full p-2"
+              style="box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1), 0px 1px 2px rgba(0, 0, 0, 0.06);"
+            />
+          </a>
+          <a href="https://twitter.com/WeDancePlatform" target="_blank">
+            <TIcon
+              name="twitter"
+              size="10"
+              class="rounded-full p-2"
+              style="box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1), 0px 1px 2px rgba(0, 0, 0, 0.06);"
+            />
+          </a>
+          <a
+            href="https://www.facebook.com/people/WeDance/100079633097491/"
+            target="_blank"
+          >
+            <TIcon
+              name="facebook"
+              size="10"
+              class="rounded-full p-2"
+              style="box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1), 0px 1px 2px rgba(0, 0, 0, 0.06);"
+            />
+          </a>
+        </div>
+      </div>
+    </section>
+  </div>
+  <div v-else>
+    Page have username
+  </div>
+</template>
+
+<script>
+import { db } from '~/plugins/firebase'
+import { useDoc } from '~/use/doc'
+import { useAuth } from '~/use/auth'
+
+export default {
+  layout: 'empty',
+  async asyncData({ redirect }) {
+    const { softUpdate } = useDoc('nfc-card')
+    let username = ''
+
+    let pageId = $nuxt.$route.params.landing
```

Comment:
`landing` is confusing name for param, rename it to `id`

```suggestion
    let pageId = $nuxt.$route.params.id
```

Reply:
It is "id" now